repos/os/src/server/vmm/gic.h

```diff
@@ -151,6 +151,8 @@ class Vmm::Gic : public Vmm::Mmio_device
 					Mmio_register gicr_igroupr0 { "GICR_IGROUPR0", Mmio_register::RO,
 					                              0x10080, 4, 0 };
 
+					virtual ~Redistributor() = default;
+
 					struct Gicr_isenabler0 : Irq_reg
```

Comment:
We usually inherit the `Interface` (https://github.com/genodelabs/genode/blob/master/repos/base/include/util/interface.h) in cases like this instead of providing a virtual destructor. It should have the same effect but is easier on the eyes.

@tworaz could you give this a try with clang?

Reply:
@nfeske Inheriting Genode::Interface works like charm. Should I push fixed patch as a separate PR, or just force push this patchset with the problematic patch amended?

On a slightly related note yesterdays checkout of clang 12 also warns that Irq_reg::read and Irq_reg::write functions defined in the same file on lines 107 & 108 are missing override keywords. I'll push a separate PR for this.

Reply:
A force push from your side would be very convenient. ;-)

Reply:
Done. I've also amended commit description to reflect the new contents of the patch.

Reply:
Thank you. I cherry-picked your commit to staging.